protocols/raft/src/main/java/io/atomix/protocols/raft/roles/AbstractAppender.java

```diff
@@ -220,7 +220,8 @@ protected void failAttempt(RaftMemberContext member, RaftRequest request, Throwa
     // when attempting to send entries to down followers.
     final int failures = member.incrementFailureCount();
     if (failures <= 3 || failures % 100 == 0) {
-      log.info("{} to {} failed: {}", request, member.getMember().memberId(), error.getMessage());
+      log.warn(
+          "{} to {} failed: {}", request, member.getMember().memberId(), error.getMessage(), error);
```

Comment:
Do we need both `error.getMessage()` and `error`?

Reply:
Probably not. Just wanted to see also stacktrace etc. I dont like printing only the message. I can remove the message thing

Reply:
:+1:

Reply:
updated branch